spec/services/idv/phone_step_spec.rb

```diff
@@ -116,27 +118,26 @@
     it 'increments step attempts' do
       expect do
         subject.submit(phone: bad_phone)
-        expect(subject.async_state.done?).to eq true
-        subject.async_state_done(subject.async_state)
       end.to(change { throttle.fetch_state!.attempts }.by(1))
     end
 
-    it 'does not increment step attempts when the vendor request times out' do
-      expect { subject.submit(phone: timeout_phone) }.to_not change { throttle.attempts }
+    it 'increments step attempts when the vendor request times out' do
+      expect do
+        subject.submit(phone: timeout_phone)
+      end.to(change { throttle.fetch_state!.attempts }.by(1))
     end
 
     it 'does not increment step attempts when the vendor raises an exception' do
```

Comment:
your test shows that it _is_ incrementing the attempts

Reply:
This is a side effect of moving the `throttle.increment!` call before the vendor request is made. 

Previously, if the request to the vendor timed out, we wouldn't count that against the user's throttle. But now we don't know if the request will time out before we make it, so the user could get throttled on timed out requests.